Makefile

```
@@ -752,7 +752,7 @@ install.tools: .install.goimports .install.md2man .install.ginkgo .install.golan
 
 .install.goimports: .gopathok
 	if [ ! -x "$(GOBIN)/goimports" ]; then \
-		$(call go-get,golang.org/x/tools/cmd/goimports); \
+		env GO111MODULE=on go get -u golang.org/x/tools/cmd/goimports@5214f41; \
```

```suggestion
		go install golang.org/x/tools/cmd/goimports; \
```
That did the trick in an F33 container for me.

---

The problem is without a tag it can/will break again. If it is only needed for another month it may be good enough though. It should be possible to use `@tag` with go install.

I just want to point out that https://github.com/containers/podman/pull/15883 says go install is not possible.

---

Thanks, I'll give it a try.  My concern though is we need to pin the package to a version because "latest" was causing problems.  The `5214f41` commit is right before the problematic change.

---

This seems to work for F33, but now F34 complains:

```
if [ ! -x "/var/tmp/go/bin/goimports" ]; then \
	go install golang.org/x/tools/cmd/goimports; \
fi
go: downloading golang.org/x/tools v0.0.0-20210106214847-113979e3529a
/var/tmp/go/pkg/mod/golang.org/x/tools@v0.0.0-20210106214847-113979e3529a/internal/imports/mod.go:21:2: missing go.sum entry for module providing package golang.org/x/mod/module (imported by golang.org/x/tools/internal/imports); to add:
	go get golang.org/x/tools/internal/imports@v0.0.0-20210106214847-113979e3529a
/var/tmp/go/pkg/mod/golang.org/x/tools@v0.0.0-20210106214847-113979e3529a/internal/gocommand/vendor.go:16:2: missing go.sum entry for module providing package golang.org/x/mod/semver (imported by golang.org/x/tools/internal/gocommand); to add:
	go get golang.org/x/tools/internal/gocommand@v0.0.0-20210106214847-113979e3529a
make: *** [Makefile:754: .install.goimports] Error 1

Exit status: 2
```

Do I just need to run `make vendor-in-container` on F34?

---

Ahh, I think I got it.  It seems `go install golang.org/x/tools/cmd/goimports@5214f41` works.

---

yeah because different go versions, this stuff is all eol for a long time.
I don't think we need this? I suggest to just remove it like https://github.com/containers/podman/pull/15883

---

Okay, that'll be my next attempt if my horrible hack doesn't work.  This branch is dying in less than a month.

---

Hmmm...actually I need a fix for `v3.0.1-rhel` also, so maybe it's worth nuking it entirely.